boilerplate/openshift/golang-osd-operator/csv-generate/catalog-build.sh

```diff
@@ -65,6 +65,7 @@ EOF
 cat <<EOF > $DOCKERFILE_REGISTRY
 FROM quay.io/openshift/origin-operator-registry:4.8.0
 COPY $SAAS_OPERATOR_DIR manifests
+RUN pip3 install --upgrade urllib3==1.26.9
```

Comment:
Is `--upgrade` appropriate when a specific version is given?

Reply:
When I tested locally it worked just fine, --upgrade may not be needed and can just do pip3 install urllib3==1.26.9

Reply:
Yeah, I'd say if it's not needed let's just remove the `--upgrade` option.